projects/i18n/switch/language-switcher-token.ts

```diff
@@ -0,0 +1,19 @@
+import type {Provider} from '@angular/core';
```

Comment:
I feel like this file should be called `language-switcher.providers.ts` and `TUI_LANGUAGE_CHUNK_LOADER` can be moved here, I don't see a point in splitting these entities into 2 files.

Reply:
fixed